libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable, Injector } from '@angular/core'
+import { SearchFilters } from '@geonetwork-ui/util/shared'
+import {
+  AbstractSearchField,
+  FieldValue,
+  FullTextSearchField,
+  IsSpatialSearchField,
+  SimpleSearchField,
+  TopicSearchField,
+} from './fields'
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FieldsService {
+  private fields = {
+    publisher: new SimpleSearchField('OrgForResource', 'asc', this.injector),
```

Comment:
Wondering if we should not rather expose the ordering as an input of the filter-dropdown component? Btw, do you think an `enum` for the orders is too verbose?

Reply:
If we don't need to customize the order, then we don't have to expose it. Alternatively we could have specific configurations for each search field in the future, but that could be provided in a different way than just an input, because this input will have to be carried across several layers. This all depends on the intended use cases.

As for enums, I think when they are sufficiently explicit strings (`'asc'` and `'desc'` for instance) I think there's really no need in using TS enums, and string unions are just fine. Also enums do not exist in Javascript, they are Typescript constructs and that make them a bit awkward IMO, so I try to avoid using them whenever possible.